astronomer/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Dict
+
+from botocore.exceptions import ClientError
+
+from astronomer.providers.amazon.aws.hooks.base_aws import AwsBaseHookAsync
+from astronomer.providers.amazon.aws.hooks.s3 import S3HookAsync
+
+
+class SageMakerHookAsync(AwsBaseHookAsync):
+    """
+    Interact with Amazon SageMaker async using aiobotocore python library.
+
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+    are passed down to the underlying AwsBaseHookAsync.
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any):
+        kwargs["client_type"] = "sagemaker"
+        super().__init__(*args, **kwargs)
+        self.s3_hook = S3HookAsync(aws_conn_id=self.aws_conn_id)
+
+    async def describe_transform_job_async(self, job_name: str) -> Dict[str, Any]:
+        """
+        Return the transform job info associated with the name
+
+        :param job_name: the name of the transform job
+        """
+        async with await self.get_client_async() as client:
+            try:
```

Comment:
Remove try-catch since you are catching and raising

Reply:
Fixed